android/java/org/chromium/chrome/browser/settings/BraveSyncScreensPreference.java

```diff
@@ -64,6 +62,7 @@
 import org.chromium.chrome.R;
 import org.chromium.chrome.browser.BraveSyncWorker;
 import org.chromium.chrome.browser.back_press.BackPressHelper;
+import org.chromium.chrome.browser.crypto_wallet.util.Utils;
```

Comment:
I got the feedback that it is not the best way to use class from `crypto_wallet` at the sync

Reply:
Modified at https://github.com/brave/brave-core/pull/30280/commits/1a55af52e63e05dbc715a23441acd1ba8281976d